tasks/dogstatsd.py

```diff
@@ -58,6 +58,14 @@ def build(ctx, rebuild=False, race=False, static=False, build_include=None,
     }
     ctx.run(cmd.format(**args), env=env)
 
+    # Manually strip the binary as go 1.10 does not pass the ldflag correctly
```

Comment:
we weren't prepared to strip all the binaries without being sure of the consequences; hence increasing the binary size above.